docker-compose.yml

```diff
@@ -1,41 +1,32 @@
 version: '3'
 services:
-  zookeeper:
-    image: wurstmeister/zookeeper
-    ports:
-      - "2181:2181"
-    logging:
-      options:
-        max-size: "10m"
-        max-file: "5"
-
   kafka:
-    image: wurstmeister/kafka
+    image: bitnami/kafka:latest
```

Comment:
Kafka is exiting & showing the following output when spun up locally using docker-compose:

> kafka-1  | kafka 16:21:15.65 INFO  ==> 
kafka-1  | kafka 16:21:15.65 INFO  ==> Welcome to the Bitnami kafka container
kafka-1  | kafka 16:21:15.65 INFO  ==> Subscribe to project updates by watching https://github.com/bitnami/containers       
kafka-1  | kafka 16:21:15.66 INFO  ==> Submit issues and feature requests at https://github.com/bitnami/containers/issues   
kafka-1  | kafka 16:21:15.66 INFO  ==>
kafka-1  | kafka 16:21:15.67 INFO  ==> ** Starting Kafka setup **
kafka-1  | mkdir: cannot create directory '/bitnami/kafka': Permission denied
kafka-1 exited with code 1

Reply:
This was occurring due to spinning up the project in a Windows command prompt while having the WSL path specified for the DOCKER_SHARED_VOLUME environment variable. When run in WSL, this does not occur.